### PR TITLE
kube-apiserver-healthcheck: actually enable on 1.17

### PR DESCRIPTION
**Pull request by justinsb**
We cherry picked, but now we need to activate the feature.

### CODE REVIEW

**Comment by justinsb** [comment]
I expect this to fail until we merge #9096

File: pkg/model/components/apiserver.go [hunk 1]
```diff
@@ -296,7 +296,7 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 		c.AnonymousAuth = fi.Bool(false)
 	}
 
-	if b.IsKubernetesGTE("1.18") {
+	if b.IsKubernetesGTE("1.17") {
```
Comment (johngmyers) [inline pkg/model/components/apiserver.go:299]: You'll want to change this on master branch as well. How about first landing to master and then cherrypicking?

Comment (mikesplain) [reply pkg/model/components/apiserver.go:299]: Fair point, that's an easy follow up.

Comment (justinsb) [reply pkg/model/components/apiserver.go:299]: Oops - will send PR to master now :-)

**Pull request merged**